get-started/part2.md

```diff
@@ -138,6 +138,12 @@ At this point, you've successfully built an image, performed a simple containeri
 
 [On to Part 3 >>](part3.md){: class="button outline-btn" style="margin-bottom: 30px; margin-right: 100%"}
 
+## Deploying to the cloud
+
+If you want to go ahead and get your containers running in either Azure or AWS then check out our blog posts on getting started with Cloud deployments and Docker
```

Comment:
How about - To run your containers in either Azure or AWS, check out our docs on getting started with cloud deployments.

Reply:
updated wording to match recommendation